### ISSUE TITLE
godoc.org shows docs for v2 (presumably)

### ISSUE THREAD

**Issue by dnutels** [segmentio/kafka-go#247]
**Describe the bug**
https://godoc.org/github.com/segmentio/kafka-go shows a version of documentation that can't actually be installed (at least not with Go 1.11 Modules)

This is what is shown in the doc:

```
type Message struct {
    // Topic is reads only and MUST NOT be set when writing messages
    Topic string

    // Partition is reads only and MUST NOT be set when writing messages
    Partition int
    Offset    int64
    Key       []byte
    Value     []byte
    Headers   []Header

    // If not set at the creation, Time will be automatically set when
    // writing the message.
    Time time.Time
}
```

And this is what gets isntalled (when importing `github.com/segmentio/kafka-go`):

```
type Message struct {
    // Topic is reads only and MUST NOT be set when writing messages
    Topic string

    // Partition is reads only and MUST NOT be set when writing messages
    Partition int
    Offset    int64
    Key       []byte
    Value     []byte

    // If not set at the creation, Time will be automatically set when
    // writing the message.
    Time time.Time
}
```

**Kafka Version**
2.1.0

**To Reproduce**
Use `kafka.Message`

**Expected behavior**
Either a documentation update to reflect the gap or a documentation update on how to install the version that is reflected in godoc.org

**Additional context**
I am absolutely open to me not properly understanding how `kafka-go` should be consumed within the Go Modules context.

**Comment by achille-roussel** [comment]
@dnutels thanks for raising the issue.

This is because we haven't tagged a release yet, the `Headers` field was introduced recently.

You can still force a dependency on a branch or specific commit with `go get github.com/segmentio/kafka-go@<commit>` I think.

Let me know if that helps, we should tag a new release soon as well.

**Comment by dnutels** [comment]
@achille-roussel 

Thank you, for your response.

Yes, running `go get github.com/segmentio/kafka-go@master` within the module installs the latest (well, master), and `go mod tidy` updates `go.mod` properly.

This, of course, is super fragile, depending on your push-to-master policies.

Do you have an ETA on the release? Or, alternatively, could you point me to the specific commit, so that I am more locked and shielded from master changes?

**Comment by achille-roussel** [comment]
We've tagged a new released (v0.2.3) so modules and doc should now be in sync.

Thanks for reporting the issue!
